src/displayBackend/wayland/WaylandZCopy.cpp

```diff
@@ -317,4 +317,111 @@ void WaylandKms::release()
 	}
 }
 
+/*******************************************************************************
+ * Linux dmabuf
+ ******************************************************************************/
+
+WaylandLinuxDmabuf::WaylandLinuxDmabuf(wl_registry* registry,
+									   uint32_t id, uint32_t version) :
+	WaylandZCopy(registry, id, version),
+	mWlLinuxDmabuf(nullptr)
+{
+	try
+	{
+		init(version);
+	}
+	catch(const std::exception& e)
+	{
+		release();
+
+		throw;
+	}
+}
+
+WaylandLinuxDmabuf::~WaylandLinuxDmabuf()
+{
+	release();
+}
+
+/*******************************************************************************
+ * Public
+ ******************************************************************************/
+
+FrameBufferPtr
+WaylandLinuxDmabuf::createLinuxDmabufBuffer(DisplayBufferPtr displayBuffer,
```

Comment:
function creates nothing , is it correct ?

Reply:
This is a patch in the middle, e.g. I am preparing infrastructure for the further implementation: stub